Formula/minimal-racket.rb

```diff
@@ -108,10 +110,64 @@ def caveats
     # ensure Homebrew openssl is used
     if OS.mac?
       output = shell_output("DYLD_PRINT_LIBRARIES=1 #{bin}/racket -e '(require openssl)' 2>&1")
-      assert_match(%r{.*openssl@1\.1/.*/libssl.*\.dylib}, output)
+      assert_match(%r{.*openssl@3/.*/libssl.*\.dylib}, output)
     else
       output = shell_output("LD_DEBUG=libs #{bin}/racket -e '(require openssl)' 2>&1")
-      assert_match "init: #{Formula["openssl@1.1"].opt_lib}/#{shared_library("libssl")}", output
+      assert_match "init: #{Formula["openssl@3"].opt_lib}/#{shared_library("libssl")}", output
     end
   end
 end
+
+__END__
+diff --git a/racket/collects/openssl/libcrypto.rkt b/racket/collects/openssl/libcrypto.rkt
+index a10c609118..afe5f96622 100644
+--- a/racket/collects/openssl/libcrypto.rkt
++++ b/racket/collects/openssl/libcrypto.rkt
+@@ -38,7 +38,10 @@
+ (define openssl-lib-versions
+   (let
+     ([versions
+-      '("1.1"
++      '(;; OpenSSL 3 works for most functionality:
```

Comment:
What does this mean by "most" functionality?

Reply:
https://racket.discourse.group/t/ubuntu-22-04-libssl-3-migration/942/2 mentions some tests failed.

Reply:
I run the tests and all [openssl tests](https://github.com/racket/racket/tree/master/pkgs/racket-test/tests/openssl) including the three tests mflatt mentioned on discourse passed.